torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -253,6 +253,7 @@ def should_skip_lowering(op: torch.fx.node.Node, qconfig_map: Dict[str, QConfigA
 #   2) The replacement static quantized module class for lowering
 STATIC_LOWER_FUSED_MODULE_MAP: Dict[Type[nn.Module], Tuple[Type[nn.Module], Type[WeightedQuantizedModule]]] = {
     nni.LinearReLU: (nnqr.Linear, nniq.LinearReLU),
+    nni.LinearLeakyReLU: (nnqr.Linear, nniq.LinearLeakyReLU),
```

Comment:
I guess this change is related to lowering, but currently it is not tested, we should add a test that goes through PTQ flow of fx graph mode quantization and lower the linear - leaky_relu pattern, and probably another test for (linear - bn - leaky_relu)

Reply:
I have added a test case in `test/quantization/fx/test_quantize_fx.py` as `TestQuantizeFx.test_linear_leaky_relu_lowering`

Reply:
do we also fuse this for fbgemm and qnnpack as well? should we skip that for these other backends?

Reply:
No. We don't fuse this for other backends. I have added a test case to confirm the pattern is not fused by default.

Reply:
For fbgemm and qnnpack, `linear + leaky_relu` are not fused and they are lowered separately. If users specify `backend='onednn'` they still need to use onednn's backend config explicitly for `prepare_fx` and `convert_fx` to fuse and lower `linear-leaky_relu`. So, a new lowering function is not needed.